src/screens/Voucher/VoucherHomeScreen.tsx

```diff
@@ -37,11 +50,40 @@ export const VoucherHomeScreen = ({ navigation, route }: StackScreenProps<any>)
   // refresh if the refreshAuth param changed, which happens after login
   useEffect(refreshAuth, [route.params?.refreshAuth]);
 
+  const { mutate: mutateLogIn } = useMutation(logIn);
+
+  useEffect(() => {
+    const accountCheck = async () => {
+      const loginData = await voucherMemberLoginInfo();
+      const savedDate = (await readFromStore(SAVED_DATE)) || moment().format('YYYY-MM-DD');
+
+      if (loginData && !moment().isSame(savedDate, 'day')) {
+        mutateLogIn(JSON.parse(loginData as string) as VoucherLogin, {
+          onSuccess: (responseData) => {
+            if (!responseData?.member) {
+              storeVoucherAuthToken();
+              storeVoucherMemberId();
+              storeVoucherMemberLoginInfo();
+              refresh();
+            }
+          }
+        });
+
+        addToStore(SAVED_DATE, moment().format('YYYY-MM-DD'));
+        setAccountCheckLoading(false);
+      } else {
+        setAccountCheckLoading(false);
+      }
```

Comment:
`setAccountCheckLoading(false);` does not need to exist twice. we can get rid of the `else` and call 0is after the if in any case.